cypress/support/pages/checkout-page.ts

```diff
@@ -1,13 +1,34 @@
 import { CHECKOUT_ELEMENTS } from "../../elements/checkout/checkout-page";
 import { DUMMY_PAYMENT } from "../../elements/checkout/dummy-payment-page";
+import { ADYEN_PAYMENT } from "../../elements/checkout/adyen-payment-page";
 
 export function payByDummyPayment() {
-  cy.addAliasToGraphRequest("order")
-    .get(CHECKOUT_ELEMENTS.paymentMethods)
-    .children()
-    .find("[value='dummy']")
-    .click()
-    .wait(2000);
+  cy.addAliasToGraphRequest("order");
+  cy.get(CHECKOUT_ELEMENTS.paymentProviders).children().find("[value='dummy']").click().wait(2000);
```

Comment:
```suggestion
  cy.get(CHECKOUT_ELEMENTS.paymentProviders).find("[value='dummy']").click().wait(2000);
```
is 'children' needed?

Reply:
Hard to say, It should work without children() but I don't have the environment and capacity to check this now. As the main goal was to fix those tests for the staging demo where we have only Adyen.